ui/theming/compose/src/main/kotlin/com/edricchan/studybuddy/ui/theming/compose/theme/compat/Color.kt

```diff
@@ -0,0 +1,18 @@
+package com.edricchan.studybuddy.ui.theming.compose.theme.compat
+
+import androidx.compose.material3.darkColorScheme
+import androidx.compose.material3.lightColorScheme
+import androidx.compose.ui.graphics.Color
+
+val StudyBuddyCompatLightColors = lightColorScheme(
+    // TODO: Add colours
+    primary = Color(0xff3948ab),
+    primaryContainer = Color(0xff00227b),
+    secondary = Color(0xffff4081)
+)
+
+val StudyBuddyCompatDarkColors = darkColorScheme(
+    primary = Color(0xff8e24aa),
```

Comment:
I'd probably suggest having these as variables in your `Color.kt` file personally

Reply:
I'll probably nuke these colours once I fully convert the whole codebase to Compose tbh 🤔